src/unified_diff.rs

```diff
@@ -54,8 +57,9 @@ where
     T: Hash + Eq + Display,
 {
     /// Create a new `UnifiedDiffBuilder` for the given `input`,
+    /// displaying `context_size` lines of context around each change,
```

Comment:
Now that I see this, I feel that `None` (and the default of 3) should be documented. But then this has to be copy-pasted as well.

What about adding a `ContextSize` type that can be instantiated with a number, has `default()`, but can in future also be extended with options to control the before/after portions of the context individually?

That way, the docs could be on the `Struct` or `Enum`, and they could be extended as well.

Reply:
Pushed some changes. Not sure if this is what you had in mind nor if it's very idiomatic. Let me know.